test/src/rabbit_auth_backend_ldap_test.erl

```diff
@@ -172,20 +172,31 @@ login() ->
           {LGood, FilterList, L}  <- logins(),
           {N, {EnvGood, Env}}     <- login_envs()]).
 
+logins() -> [logins_direct() | logins_network()].
+
 %% Format for login tests, {Outcome, FilterList, Login}.
 %% Tests skipped for each login_env reference in FilterList.
-logins() ->
-    [{bad,  [5], #amqp_params_network{}},
-     {bad,  [5], #amqp_params_network{username     = << ?ALICE_NAME >>}},
-     {bad,  [5], #amqp_params_network{username     = << ?ALICE_NAME >>,
-                                      password     = <<"password">>}},
-     {bad,  [5], #amqp_params_network{username     = <<"Alice">>,
-                                      password     = <<"Alicja">>,
-                                      virtual_host = << ?VHOST >>}},
-     {bad,  [1, 2, 3, 4, 6], ?CAROL},
-     {good, [5], ?ALICE},
-     {good, [5], ?BOB},
-     {good, [1, 2, 3, 4, 6], ?PETER}].
+logins_network() ->
+    [{bad,  [5, 7], #amqp_params_network{}},
+     {bad,  [5, 7], #amqp_params_network{username     = <<?ALICE_NAME>>}},
+     {bad,  [5, 7], #amqp_params_network{username     = <<?ALICE_NAME>>,
+                                         password     = <<"password">>}},
+     {bad,  [5, 7], #amqp_params_network{username     = <<"Alice">>,
+                                         password     = <<"Alicja">>,
+                                         virtual_host = <<?VHOST>>}},
+     {bad,  [1, 2, 3, 4, 6, 7], ?CAROL},
+     {good, [5, 7], ?ALICE},
+     {good, [5, 7], ?BOB},
+     {good, [1, 2, 3, 4, 6, 7], ?PETER}].
+
+logins_direct() ->
+    [{bad,  [5], #amqp_params_direct{}},
+     {bad,  [5], #amqp_params_direct{username      = <<?ALICE_NAME>>}},
+     {bad,  [5], #amqp_params_direct{username      = <<?ALICE_NAME>>,
+                                     password      = <<"password">>}},
+     {good, [5], #amqp_params_direct{username      = <<?ALICE_NAME>>,
```

Comment:
@Ayanda-D this test passes without https://github.com/rabbitmq/rabbitmq-server/pull/722 
Is it correct?